src/modelfree/score_agent.py

```diff
@@ -50,6 +58,33 @@ def get_empirical_score(_run, env, agents, episodes, render=False):
     return result
 
 
+def _clean_video_directory_structure(observer_obj):
+    """
+    A simple utility method to take saved videos within a Sacred run structure and clean
+    up the file pathways, so that all videos are organized under a "videos" directory,
+    and within that organized by environment ID.
+
+    :param observer_obj: A Sacred FileStorageObserver object
+    :return:
+    """
+    basedir = observer_obj.dir
+    video_files = glob.glob("{}/*.mp4".format(basedir))
+    if len(video_files) == 0:
+        return
+
+    ptn = re.compile(r'env_\d*_video.(\d*)')
+    new_video_dir = os.path.join(basedir, "videos")
+    os.mkdir(new_video_dir)
+
+    for video_file in video_files:
+        search_result = ptn.search(video_file)
+        if search_result is None:
+            continue
+        episode_id = search_result.groups()[0]
+        new_file_name = "episode_{}_recording.mp4".format(int(episode_id))
+        os.rename(video_file, os.path.join(new_video_dir, new_file_name))
```

Comment:
Episode IDs won't be unique so I think this is missing some logic. Maybe number it as `env_id * num_envs + episode_id`, or just number them sequentially...?

Also, should we copy over the `*.meta.json` file as well? Sometimes we stick useful fields in there, e.g. `winners`.

Reply:
Ah, good point. I'm a little uncomfortable about inferring an ordered/sequential episode numbering in the multi env case where there isn't actually a natural/guaranteed ordering. I think I'd just prefer to add the env ID to the file name